### PR TITLE
Breaking out settings pages

### PR DESCRIPTION
**Pull request by aleib**

### CODE REVIEW

File: docs/source/dashboard/configuration/wifi_page.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
+Wi-Fi
+=====
+
+Configuration options when setting up a new SSID
+
+.. _ssid-setting:
```
Comment (clangenhoven) [inline docs/source/dashboard/configuration/wifi_page.rst]: Are you planning on adding refs for these labels anywhere?

Comment (aleib) [reply docs/source/dashboard/configuration/wifi_page.rst]: they appear in the breakdown so figured no need

Comment (aleib) [reply docs/source/dashboard/configuration/wifi_page.rst]: image

Comment (aleib) [reply docs/source/dashboard/configuration/wifi_page.rst]: (using the links)

**Pull request merged**